data_diff/dbt_parser.py

```diff
@@ -68,7 +86,60 @@ def get_datadiff_variables(self) -> dict:
         vars = get_from_dict_with_raise(self.project_dict, "vars", error_message)
         return get_from_dict_with_raise(vars, "data_diff", error_message)
 
-    def get_models(self):
+    def get_models(self, dbt_selection: Optional[str] = None):
+        dbt_version = parse_version(self.dbt_version)
+        if dbt_selection:
+            if dbt_version.major == 1 and dbt_version.minor >= 5:
```

Comment:
**nit:** Despite this works now, it would break with v2.x, when we forget about such version restrictions. A somewhat better and common approach is lexographical comparison of tuples:

```suggestion
            if (dbt_version.major, dbt_version.minor) >= (1, 5):
```